src/easy/handle.rs

```diff
@@ -104,6 +104,7 @@ pub struct EasyData {
 unsafe impl Send for EasyData {}
 
 #[derive(Default)]
+#[allow(clippy::type_complexity)]
```

Comment:
I would personally prefer to not have lint annotation for clippy lints, so can this and others be removed?